apps/web/src/components/NotificationDialog/UnreadAlertBellIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
I think you can remove it

Reply:
Sadly we can't, Typescript is confused because you can have automatic JSX support so React is not requried just for JSX but we don't have that so it technically is.

Eslint properly identifies this but TS does not and doesn;'t see us using react so fades it out but we technically need it for JSX :sweat_smile: 

Maybe we can look into auto JSX support so we don't need to import react in every file manually :thinking: